File: RepoTransBench/target_projects/Java/C#/mohak1712_UberUX/tests/original/LoginActivityTest.cs
using Moq;
using Xunit;

namespace UberUX.Tests.Original
{
    public class LoginActivityTest
    {
        private UberUX.LoginActivity activity;

        public LoginActivityTest()
        {
            var mock = new Mock<UberUX.LoginActivity> { CallBase = true };
            activity = mock.Object;
        }

        [Fact]
        public void TestOnCreateExecutesWithoutCrash()
        {
            var bundle = new object(); // stand-in for Bundle
            activity.onCreate(bundle);
            Assert.NotNull(activity);
        }

        [Fact]
        public void TestSetupWindowAnimationsNoCrash()
        {
            var mock = Mock.Get(activity);
            mock.Protected().Setup("setupWindowAnimations");
            activity.GetType().GetMethod("setupWindowAnimations", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance)
                ?.Invoke(activity, null);
            Assert.NotNull(activity);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/mohak1712_UberUX/tests/original/login_activity_test.go
package original

import (
	"testing"

	"mohak1712_uberux/tests"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/mock"
)

func TestLoginActivityOnCreateExecutesWithoutCrash(t *testing.T) {
	l := new(tests.LoginActivity)
	l.On("OnCreate", mock.Anything).Return()
	bundle := tests.Bundle{}
	l.OnCreate(bundle)
	assert.NotNil(t, l)
}

func TestLoginActivitySetupWindowAnimationsNoCrash(t *testing.T) {
	l := new(tests.LoginActivity)
	l.On("SetupWindowAnimations").Return()
	l.SetupWindowAnimations()
}